SPECS/kata-containers-cc/mariner-coco-build-uvm-image.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Assuming we wrote this, please add the standard Microsoft OSS copyright headers. Applies throughout

Reply:
Copyrights added for both scripts